keyword/func.go

```diff
@@ -33,6 +33,7 @@ const (
 	Localtimestamp        = "LOCALTIMESTAMP"
 	Makedate              = "MAKEDATE"
 	Maketime              = "MAKETIME"
+	Max                   = "MAX"
```

Comment:
Here is the block for the date and time functions. Please add this line with comments on the new line.

```go
    // Numeric functions
    Max = "MAX"
```

Reply:
Fixed